include/ppx/metrics.h

```diff
@@ -172,7 +186,7 @@ class MetricGauge final : public Metric
 private:
     MetricMetadata               mMetadata;
     std::vector<TimeSeriesEntry> mTimeSeries;
-    Stats                        mBasicStats;
+    Stats                        mStats;
```

Comment:
I think it was storing only the basic stats in this struct before, since you are splitting the Stats in two parts, one being the basic stats, should this just be GaugeBasicStatistics instead? And then we can actually construct Stats later. Otherwise it sounds like the complex stats are just not valid in this struct

Reply:
the original also stores both but the name was named as mBasicStats, but I do agree with you that there is no point of storing the complex here. Changed as suggested